### PR TITLE
Fix buttons in shadowroot not being captured (resolves PostHog/posthog#1469)

### PR DESCRIPTION
**Pull request by timgl**
## Changes

Previously we weren't capturing clicks within shadowroots. See playground/flags/demo.html for an example testcase.

## Checklist
- [x] Tests for new code (if applicable)
- [ ] TypeScript definitions (module.d.ts) updated and in sync with library exports (if applicable)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
         "lint": "eslint src tests --fix",
         "integration-test": "echo 'Browse to localhost:3000/tests' && node testServer.js",
         "prepublish": "yarn test && yarn build && yarn build-module && yarn process-types",
-        "test": "jest",
+        "test": "nodemon --ext js --exec 'yarn jest'",
```
Comment (Twixes) [inline package.json:17]: Hm, wouldn't `jest` with the `--watch` options do the job here?

Comment (timgl) [reply package.json:17]: Ah, I didn't know that was an option! Updated

File: playground/flags/demo.html [hunk 2]
```diff
@@ -23,9 +23,16 @@ <h2>Demo site</h2>
     <div class='asdlfkjj asdlfkj asdflkj asdflkj asdf'>
         <a href='https://posthog.com'>Here's a link to the outside world</a>
     </div>
+    <div id="shadow" ></div>
```
Comment (Twixes) [inline playground/flags/demo.html:26]: This seems unused.

**Review by Twixes — APPROVED** [review]
Interestingly this enhancement will cause using the Toolbar to now capture events like any other clickable elements on the user's website (as it's all under a shadow root). I guess setting `className="ph-no-capture"` on it will prevent that from happening.

Screen Shot 2020-08-25 at 17 07 09

**Pull request merged**